_sass/blocks/_downloads.scss

```diff
@@ -3,7 +3,7 @@
 .downloads-download_links {
   a {
     font-size: 1rem;
-    font-weight: $weight-book;
+    font-weight: 400;
```

Comment:
`normal` or `400` is the default font-weight, so unless we are trying to override another style, I think this declaration is redundant. If we are trying to override something, we can probably make another variable for resetting a font to normal? Or is this only used here?

If we do need to reset the weight, and we don't think a variable is necessary, we should leave a comment explaining why we are manually setting the default weight.